### PR TITLE
feat(plasma-colors): Update colors palette

### PR DESCRIPTION
**Pull request by neretin-trike**
Обновлена основная цветовая палитра согласна новым макетам.

Добавлены фолбэки для сохранения обратной совместимости.

:baby_chick: Download canary assets:

[color_sberHealth_ios-swift--canary.509.5004758697.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.509.5004758697.swift)  
[color_sberHealth_kotlin--canary.509.5004758697.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.509.5004758697.kt)  
[color_sberHealth_react-native--canary.509.5004758697.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.509.5004758697.ts)  
[color_sbermarket_business_ios-swift--canary.509.5004758697.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.509.5004758697.swift)  
[color_sbermarket_business_kotlin--canary.509.5004758697.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.509.5004758697.kt)  
[color_sbermarket_business_react-native--canary.509.5004758697.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.509.5004758697.ts)  
[color_sbermarket_ios-swift--canary.509.5004758697.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.509.5004758697.swift)  
[color_sbermarket_kotlin--canary.509.5004758697.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.509.5004758697.kt)  
[color_sbermarket_metro_ios-swift--canary.509.5004758697.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.509.5004758697.swift)  
[color_sbermarket_metro_kotlin--canary.509.5004758697.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.509.5004758697.kt)  
[color_sbermarket_metro_react-native--canary.509.5004758697.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.509.5004758697.ts)  
[color_sbermarket_react-native--canary.509.5004758697.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.509.5004758697.ts)  
[color_sbermarket_selgros_ios-swift--canary.509.5004758697.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.509.5004758697.swift)  
[color_sbermarket_selgros_kotlin--canary.509.5004758697.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.509.5004758697.kt)  
[color_sbermarket_selgros_react-native--canary.509.5004758697.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.509.5004758697.ts)  
[color_sbermarket_wlbusiness_ios-swift--canary.509.5004758697.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.509.5004758697.swift)  
[color_sbermarket_wlbusiness_kotlin--canary.509.5004758697.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.509.5004758697.kt)  
[color_sbermarket_wlbusiness_react-native--canary.509.5004758697.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.509.5004758697.ts)  
[color_sberprime_ios-swift--canary.509.5004758697.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.509.5004758697.swift)  
[color_sberprime_kotlin--canary.509.5004758697.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.509.5004758697.kt)  
[color_sberprime_react-native--canary.509.5004758697.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.509.5004758697.ts)  
[shadow_sbermarket_react-native--canary.509.5004758697.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.509.5004758697.ts)  
[typo_mage_ios-swift--canary.509.5004758697.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.509.5004758697.swift)  
[typo_mage_kotlin--canary.509.5004758697.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.509.5004758697.kt)  
[typo_mage_react-native--canary.509.5004758697.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.509.5004758697.ts)  
[typo_plasma_ios-swift--canary.509.5004758697.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.509.5004758697.swift)  
[typo_plasma_kotlin--canary.509.5004758697.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.509.5004758697.kt)  
[typo_plasma_react-native--canary.509.5004758697.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.509.5004758697.ts)  
[typo_ruler_ios-swift--canary.509.5004758697.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.509.5004758697.swift)  
[typo_ruler_kotlin--canary.509.5004758697.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.509.5004758697.kt)  
[typo_ruler_react-native--canary.509.5004758697.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.509.5004758697.ts)  
[typo_sage_ios-swift--canary.509.5004758697.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.509.5004758697.swift)  
[typo_sage_kotlin--canary.509.5004758697.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.509.5004758697.kt)  
[typo_sage_react-native--canary.509.5004758697.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.509.5004758697.ts)  
[typo_sbermarket_ios-swift--canary.509.5004758697.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.509.5004758697.swift)  
[typo_sbermarket_kotlin--canary.509.5004758697.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.509.5004758697.kt)  
[typo_sbermarket_react-native--canary.509.5004758697.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.509.5004758697.ts)  
[typo_soulmate_ios-swift--canary.509.5004758697.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.509.5004758697.swift)  
[typo_soulmate_kotlin--canary.509.5004758697.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.509.5004758697.kt)  
[typo_soulmate_react-native--canary.509.5004758697.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.509.5004758697.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.211.0-canary.509.5013015745.0
  npm install @salutejs/plasma-colors@0.7.0-canary.509.5013015745.0
  npm install @salutejs/plasma-core@1.116.0-canary.509.5013015745.0
  npm install @salutejs/plasma-hope@1.211.0-canary.509.5013015745.0
  npm install @salutejs/plasma-icons@1.145.0-canary.509.5013015745.0
  npm install @salutejs/plasma-temple@1.160.0-canary.509.5013015745.0
  npm install @salutejs/plasma-tokens-b2b@1.19.0-canary.509.5013015745.0
  npm install @salutejs/plasma-tokens-b2c@0.28.0-canary.509.5013015745.0
  npm install @salutejs/plasma-tokens-web@1.34.0-canary.509.5013015745.0
  npm install @salutejs/plasma-tokens@1.51.0-canary.509.5013015745.0
  npm install @salutejs/plasma-ui@1.192.0-canary.509.5013015745.0
  npm install @salutejs/plasma-web@1.211.0-canary.509.5013015745.0
  npm install @salutejs/plasma-cy-utils@0.58.0-canary.509.5013015745.0
  npm install @salutejs/plasma-sb-utils@0.114.0-canary.509.5013015745.0
  # or 
  yarn add @salutejs/plasma-b2c@1.211.0-canary.509.5013015745.0
  yarn add @salutejs/plasma-colors@0.7.0-canary.509.5013015745.0
  yarn add @salutejs/plasma-core@1.116.0-canary.509.5013015745.0
  yarn add @salutejs/plasma-hope@1.211.0-canary.509.5013015745.0
  yarn add @salutejs/plasma-icons@1.145.0-canary.509.5013015745.0
  yarn add @salutejs/plasma-temple@1.160.0-canary.509.5013015745.0
  yarn add @salutejs/plasma-tokens-b2b@1.19.0-canary.509.5013015745.0
  yarn add @salutejs/plasma-tokens-b2c@0.28.0-canary.509.5013015745.0
  yarn add @salutejs/plasma-tokens-web@1.34.0-canary.509.5013015745.0
  yarn add @salutejs/plasma-tokens@1.51.0-canary.509.5013015745.0
  yarn add @salutejs/plasma-ui@1.192.0-canary.509.5013015745.0
  yarn add @salutejs/plasma-web@1.211.0-canary.509.5013015745.0
  yarn add @salutejs/plasma-cy-utils@0.58.0-canary.509.5013015745.0
  yarn add @salutejs/plasma-sb-utils@0.114.0-canary.509.5013015745.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-509/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-509/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Review by Yakutoc — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-509/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-509/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-509/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-509/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-509/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-509/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-509/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

File: packages/plasma-tokens-b2b/data/colors.ts [hunk 1]
```diff
@@ -1,6 +1,12 @@
 import { general } from '@salutejs/plasma-colors';
 import { lightenColor } from '@salutejs/plasma-tokens-utils';
 
+general.electricBlue['400'] = '#5993FF'; // TODO: https://github.com/salute-developers/plasma/issues/508
```
Comment (Yeti-or) [inline packages/plasma-tokens-b2b/data/colors.ts:4]: я правильно понимаю, что это старые цвета которых нет больше в палитре?

**Review by Yeti-or — APPROVED** [review]

File: packages/plasma-tokens-b2b/data/colors.ts (hunk 1 above)
Comment (neretin-trike) [reply packages/plasma-tokens-b2b/data/colors.ts:4]: да, всё так

**Pull request merged**